packages/api/tests/endpoints/test-executions.js

```diff
@@ -795,3 +829,154 @@ test.serial('POST /executions/workflows-by-granules returns correct workflows wh
 
   t.deepEqual(response.body.sort(), ['fakeWorkflow', 'workflow2']);
 });
+
+test.serial('POST an api execute record creates a new execution in Dynamo and PG with correct timestamps', async (t) => {
+  const newExecution = fakeExecutionFactoryV2({
+    asyncOperationId: t.context.testAsyncOperation.id,
+    collectionId: t.context.collectionId,
+    parentArn: t.context.fakeApiExecutions[1].arn,
+  });
+
+  const expectedPgRecord = await translateApiExecutionToPostgresExecution(
+    newExecution,
+    t.context.knex
+  );
+
+  await request(app)
+    .post('/executions')
+    .send(newExecution)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  const fetchedDynamoRecord = await executionModel.get({
+    arn: newExecution.arn,
+  });
+
+  const fetchedPgRecord = await executionPgModel.get(
+    t.context.knex,
+    {
+      arn: newExecution.arn,
+    }
+  );
+
+  t.true(fetchedDynamoRecord.createdAt > newExecution.createdAt);
+  t.true(fetchedDynamoRecord.updatedAt > newExecution.updatedAt);
+
+  // PG and Dynamo records have the same timestamps
+  t.is(fetchedPgRecord.created_at.getTime(), fetchedDynamoRecord.createdAt);
+  t.is(fetchedPgRecord.updated_at.getTime(), fetchedDynamoRecord.updatedAt);
+
+  t.is(fetchedPgRecord.arn, fetchedDynamoRecord.arn);
+  t.truthy(fetchedPgRecord.cumulus_id);
+  t.truthy(fetchedPgRecord.async_operation_cumulus_id);
+  t.truthy(fetchedPgRecord.collection_cumulus_id);
+  t.truthy(fetchedPgRecord.parent_cumulus_id);
```

Comment:
We have `asyncOperationCumulusId` and `collectionCumulusId` in `t.context` so we should assert against the real expected values, not just check for truthiness. We should be able to get the parent_cumulus_id from t.context as well with some minor changes to the test setup